proto/dydxprotocol/clob/order.proto

```diff
@@ -227,3 +228,22 @@ message TransactionOrdering {
   // Within the block, the unique transaction index.
   uint32 transaction_index = 2;
 }
+
+// StreamLiquidationOrder represents an protocol-generated IOC liquidation order.
+// Used in full node streaming.
+message StreamLiquidationOrder {
+  // Information about this liquidation order.
+  PerpetualLiquidationInfo liquidation_info = 1;
+
+  // CLOB pair ID of the CLOB pair the liquidation order will be matched against.
+  uint32 clob_pair_id = 2;
+
+  // True if this is a buy order liquidating a short position, false if vice versa.
+  bool is_buy = 3;
```

Comment:
should we use `Side` similar to `Order` for consistency? 

no strong preference though

Reply:
Hmm I think it would be a little weird since we nest `Side` in the Order proto currently, and if we change it to Side for liquidation order we would have to nest it in the Liquidation order proto. I'll keep it as is for minimal changes.